cmd/openshift-dev-helpers/main.go

```diff
@@ -46,7 +46,7 @@ func NewCmdDevHelpers(streams genericclioptions.IOStreams) *cobra.Command {
 				return err
 			}
 
-			return nil
+			return c.Help()
```

Comment:
How about this @soltysh ?

Reply:
I'd prefer we properly support `--help` flag and then use wire it as a usage function so that when you don't pass arguments it'll print you one.

Reply:
okay I will try to work on this part.